news/2020-03-13-featured-artist-beatmap-updates-from-the-mappers-guild.md

```diff
@@ -0,0 +1,439 @@
+
+For those who don't know yet, the Mappers' Guild is a way for mappers with three or more ranked beatmaps to earn rewards for creating beatmaps of songs from the Featured Artist library. If you're interested in the Guild, be sure to look at the [Mappers' Guild website](https://mappersguild.com/) for more information.
+
+Recently there has also been some new developments in the Mappers' Guild system! The first is that now anyone can view the website and keep up with what Mappers' Guild members are making, so be sure to check it out if you are curious about what the mappers are working on. Next, the Guild now has solo quests available for members to take on by themselves, giving mappers the option to work alone if that's what they prefer. Lastly, with the return of the [Monthly Beatmapping Contests](https://osu.ppy.sh/help/wiki/Contests/Monthly_Beatmapping_Contest), guild members who participate can also earn points in the Mappers' Guild for doing so, which you can read more about on the linked wiki page.
+
+Today we'll be going over finished quests from the Mappers' Guild, featured artist beatmap pack  and medal updates, and a list of all ranked featured artist beatmaps since our last news post in late November. Now let's start going through all the quests completed since last update!
+
+### Navigation
+
+- [Finished quests](#finished-quests)
+- [Pack and medal updates](#pack-medal-updates)
+- [Ranked map updates](#ranked-map-updates)
+
+## <a id="finished-quests"></a>Finished quests
+
+![Aether Realm header](https://assets.ppy.sh/artists/47/header.jpg)
+
+For the **Aether Realm mini-pack (osu!taiko)** quest, the mappers had to create and rank at least 3 mapsets of songs by Aether Realm, each hosted by a different user.
+
+This quest was completed by **[Raiden](https://osu.ppy.sh/users/2239480)**, **[Greenshell](https://osu.ppy.sh/users/8693851)**, **[LZD](https://osu.ppy.sh/users/224335)** and **[frukoyurdakul](https://osu.ppy.sh/users/7612550)**
+
+- [Aether Realm - Tarot](https://osu.ppy.sh/beatmapsets/1030740#taiko/2155553) by [Raiden](https://osu.ppy.sh/users/2239480)
+- [Aether Realm - The Devil](https://osu.ppy.sh/beatmapsets/1053572#taiko/2201603) by [LZD](https://osu.ppy.sh/users/224335)
+- [Aether Realm - The Emperor](https://osu.ppy.sh/beatmapsets/1047197#taiko/2189025) by [Greenshell](https://osu.ppy.sh/users/8693851)
+
+![Carpool Tunnel header](https://assets.ppy.sh/artists/42/header.jpg)
+
+For the **Carpool Tunnel mini-pack** quests, the mappers had to create and rank at least 4 mapsets of songs by Carpool Tunnel, each hosted by a different user.
+
+The **osu!catch** version of this quest was completed by **[Jemzuu](https://osu.ppy.sh/users/7890134)**, **[Crowley](https://osu.ppy.sh/users/6341006)**, **[JierYagtama](https://osu.ppy.sh/users/7483452)**, **[Du5t](https://osu.ppy.sh/users/6053071)** and **[Dapuluous](https://osu.ppy.sh/users/8140944)**
```

Comment:
when multiple modes have completed the same quest, it'd make more sense to order them in the way people expect (osu, taiko, catch, mania)